src/Core/Language.cs

```diff
@@ -96,6 +101,19 @@ public static IEnumerable<Language> KnownLanguages
             }
         }
 
+        /// <summary>
+        /// Matches the repository name to its language object counterpart.
+        /// </summary>
+        private static readonly Dictionary<string, Language> repoKeyToLanguage = new Dictionary<string, Language>()
```

Comment:
Are we using dictionaries for these kinds of things? To me it made sense but maybe I am wrong

Reply:
Yep, makes sense.